modules/ROOT/pages/delete-identifiers.adoc

```diff
@@ -1,22 +1,28 @@
 = Delete Identifiers
 
-You might want to delete an identifier when it is no longer needed. 
+When an identifier is no longer needed, consider deleting the identifier.
 
-You can delete identifiers such as X12 ISA, X12 GS, AS2, and DUNS. You cannot delete the last identifier, as there must be at least one identifier in the list of identifiers.
+You can delete identifiers such as X12 ISA, X12 GS, AS2, and DUNS.
 
-Before you delete an identifier, consider the following: 
+Before you delete an identifier, consider the following:
 
-* Deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner.
-* Do you need to replace the identifier with another identifier? +
-If you need to replace the identifier, record the values and qualifier type needed.
-* Remember that there must always be at least one identifier per message flow before you can delete it.
+* Remember that there must always be at least one identifier per message flow.
 * You cannot delete an identifier that is used in a message flow.
+* Deleting an existing host or partner identifier impacts the identification and routing of B2B messages to the associated partner.
+* If you need to replace the identifier with another identifier, record the required values and qualifier type.
+
 
 == Deleting Identifiers
 
 . In the left navigation menu, select *Partners*.
-. From the list of partners, select the partner that has the identifier to delete. 
-. In the *Identifiers* section, hover over the identifier you want to delete and click the trash can icon that appears on right-hand side of the *Identifiers* row.
-. If you want to replace the identifier, click *New* and xref:partner-manager-identifiers.adoc[follow the steps to create an identifier]. 
+. From the list of partners, select the partner containing the identifier you want to delete.
```

Comment:
I don't think we need to specify the list; I would assume that selecting *Partners* takes me to something like a list, and the user is in the UI after all:
```suggestion
. Select the partner containing the identifier you want to delete.
```